metadata.yaml

```diff
@@ -1,29 +1,26 @@
-# This file populates the Overview on Charmhub.
-# See https://juju.is/docs/sdk/metadata-reference for a checklist and guidance.
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: identity-platform-login-ui-operator
+assumes:
+  - k8s-api
```

Comment:
I've never seen this before, is it needed? @nsklikas do you know?

Reply:
https://juju.is/docs/sdk/metadata-yaml#heading--assumes seems like a good thing to add to all of our charms